sql-db/vertx-sql/src/main/resources/application.properties

```diff
@@ -4,7 +4,7 @@ app.selected.db=postgresql
 
 # Quarkus
 quarkus.http.port=8082
-quarkus.http.test.port=8081
```

Comment:
Why do we need this change?

Reply:
You are right is not required anymore (in this scenario)